requirements_nover.txt

```diff
@@ -0,0 +1,11 @@
+certifi
```

Comment:
Depending on the thoughts of the developer/maintainer, I think it's best to version lock this and to replace the existing requirements file as otherwise both files will have to be maintained. This file will eventually fall out of sync as soon as a package makes a breaking change.

Reply:
I'm against version locking by default. If a package makes a breaking change, the particular requirement can still be version locked. Besides, I don't see no harm in two requirements files.

But I'm not trying to get the entire PR merged anyway. It's just here for Oscar's consideration.